### PR TITLE
Backport windows cmake changes

### PR DESCRIPTION
**Pull request by dnakamura**

### CODE REVIEW

**Comment by dnakamura** [comment]
Note: Might require eclipse/omr#5805 (seems to depend on if the system has had the dia SDK dlls registered properly or not)

File: jdk/make/closed/autoconf/custom-hook.m4 [hunk 1]
```diff
@@ -666,16 +670,16 @@ AC_DEFUN([CONFIGURE_OPENSSL],
 ])
 
 # Create a tool wrapper for use by cmake.
-# Consists of a shell script which wraps commands with an invocation of fixpath.
-# OPENJ9_GENERATE_TOOL_WRAPER(<name_of_wrapper>, <command_to_call>)
+# Consists of a shell script which wraps commands with an invocation of a wrapper command.
+# OPENJ9_GENERATE_TOOL_WRAPER(<name_of_output>, <name_of_wrapper>, <command_to_call>)
```
Comment (keithc-ca) [inline jdk/make/closed/autoconf/custom-hook.m4:674]: Please fix the typo ('wrapper' has two p's).

**Comment by keithc-ca** [comment]
You'll need to run `bash common/autoconf/autogen.sh` to update the generated configure scripts.

File: jdk/make/closed/autoconf/generated-configure.sh [hunk 2]
```diff
@@ -4541,16 +4545,16 @@ VS_SDK_PLATFORM_NAME_2017=
 
 
 # Create a tool wrapper for use by cmake.
-# Consists of a shell script which wraps commands with an invocation of fixpath.
-# OPENJ9_GENERATE_TOOL_WRAPER(<name_of_wrapper>, <command_to_call>)
+# Consists of a shell script which wraps commands with an invocation of a wrapper command.
+# OPENJ9_GENERATE_TOOL_WRAPER(<name_of_output>, <name_of_wrapper>, <command_to_call>)
```
Comment (keithc-ca) [inline jdk/make/closed/autoconf/generated-configure.sh:4549]: This should be regenerated to reflect changes in `custom-hook.m4`.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: jdk/make/closed/autoconf/generated-configure.sh (hunk 2 above)
Comment (dnakamura) [reply jdk/make/closed/autoconf/generated-configure.sh:4549]: oops. thought I had

**Comment by keithc-ca** [comment]
> Note: Might require [eclipse/omr#5805](https://github.com/eclipse/omr/pull/5805)

When will we have an answer to that question?

**Review by keithc-ca — APPROVED** [review]

**Comment by pshipton** [comment]
Seems we should try a PR build without that change, and if it works then this PR is sufficient for building at the OpenJ9 project itself.

**Comment by dnakamura** [comment]
Digging into it I believe the issue is related to using a different version of VS. So the patch will be required

**Comment by keithc-ca** [comment]
Jenkins test sanity win,win32 jdk8

**Comment by keithc-ca** [comment]
Jenkins test sanity win3m,win32cm jdk8

**Comment by keithc-ca** [comment]
Jenkins test sanity wincm,win32cm jdk8

**Comment by pshipton** [comment]
jenkins compile wincm jdk8 depends eclipse/omr#5805

**Comment by pshipton** [comment]
The compiles without the omr change failed.
https://ci.eclipse.org/openj9/job/Build_JDK8_x86-32_windows_cm_Personal/2/
https://ci.eclipse.org/openj9/job/Build_JDK8_x86-64_windows_cm_Personal/2/

**Comment by pshipton** [comment]
jenkins compile win32cm jdk8 depends eclipse/omr#5805

**Comment by pshipton** [comment]
The same errors occur even with the OMR changes, see the previous comment https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/477#issuecomment-777803312 or look at the latest PR builds.

https://ci.eclipse.org/openj9/job/Build_JDK8_x86-32_windows_cm_Personal/3/
https://ci.eclipse.org/openj9/job/Build_JDK8_x86-64_windows_cm_Personal/3/

**Comment by dnakamura** [comment]
the clang error would be unrelated to the omr pr, its a failure in the fixpath wrapper we put around clang, looking into it now

**Comment by dnakamura** [comment]
Pushed missing changes to fix 64 bit builds. Still looking at 32bit issues

**Comment by pshipton** [comment]
Not sure the 64-bit build needs the OMR change, trying without it. 

jenkins compile wincm jdk8

**Comment by pshipton** [comment]
jenkins compile wincm jdk8 depends eclipse/omr#5805

**Comment by pshipton** [comment]
Both PR builds succeeded.

**Comment by pshipton** [comment]
This change is sufficient for 64-bit builds so I'll merge this support.

**Pull request merged**